scripts/irods/execute.py

```diff
@@ -84,16 +84,25 @@ def execute_command_timeout(args, timeout=10, **kwargs):
             'The call {0} did not complete within'
             ' {1} seconds.'.format(args, timeout))
 
-def execute_command_permissive(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, input=None, **kwargs):
+def execute_command_permissive(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, input=None, foreground=None, **kwargs):
```

Comment:
foreground=False ?     rather than None?

Reply:
Thought I used `False` for that. Will change.

Reply:
Updated.